engine/source/engine.cpp

```diff
@@ -128,16 +129,80 @@ ModuleTickOrder OldEngine::Init(Engine& engine)
     audioModule.LoadBank(stringBank);
     audioModule.LoadBank(bi);
 
+    // Setting up game input actions
+    GameActions gameActions {};
```

Comment:
Is this entire section of setup also necessary for when working with steam input, or do the input scheme files already handle this?

Reply:
All of the actions and action sets with the correct names still needs to be set up. Since those names are used to check for Steam Input actions that exist in the Steam Input files. Then Steam checks the inputs from the controller based on the files given. It is an extra layer of indirection needed, but it adds the possibility for the user to change what inputs map to what actions via the Steam Overlay.